spec/composite-chart-spec.js

```diff
@@ -327,8 +327,8 @@ describe('dc.compositeChart', function() {
 
                 dateValueGroupLine2.on("click")(dateValueGroupLine2.datum());
                 expect(dateValueGroupLine2.text()).toBe('Date Value Group Line 2');
-                expect(d3.select(chart.selectAll('g.dc-legend g.dc-legend-item')[0][3]).classed("fadeout")).toBeTruthy();                
-                expect(chart.selectAll("path.line").size()).toEqual(3);
+                expect(dateValueGroupLine2.classed("fadeout")).toBeFalsy();
+                expect(chart.selectAll("path.line").size()).toEqual(1);
```

Comment:
@mtraynham, so.... if you have to make seemingly random changes to the tests like this, that just might a sign that the changes broke something. 

In this case, since the compositeChart `generateG()` calls `child.chartGroup(_chart.chartGroup())`, and the child charts have the same anchor as the parent, it ends up deregistering the composite chart and registering the child chart instead! So now all but one of the lines disappear when the chart rerenders.

I'll fix this up in another commit, once I figure out a good way. I guess this proves that testing is good, because it wasn't at all obvious that such a harmless-looking change could have such a disastrous effect. A little scary that this only broke one test, though.